src/server/game/Battlegrounds/Battleground.cpp

```diff
@@ -443,7 +441,7 @@ inline void Battleground::_ProcessJoin(uint32 diff)
 
         if (!FindBgMap())
         {
-            LOG_ERROR("server", "Battleground::_ProcessJoin: map (map id: %u, instance id: %u) is not created!", m_MapId, m_InstanceID);
+            LOG_ERROR("bg.battleground", "Battleground::_ProcessJoin: map (map id: %u, instance id: %u) is not created!", m_MapId, m_InstanceID);
```

Comment:
what does `bg.battleground` mean? it sounds a repetition to me as `bg` stands for `battleground`...

Reply:
because you also have `bg.arena`. Both are classed as battlegrounds.

You can find in our code many cases of `bg->isBattleground()` and `bg->isArena()`

Reply:
@Kitzunu Right

Reply:
that was what I thought but... I wish there was something more generic we could use instead of `bg.` but at the same time I can't come up with a better name myself...

Reply:
You can always change it later